stdlib/printexc.mli

```diff
@@ -96,7 +96,7 @@ type raw_backtrace
     actually printed, which may be useful if you record more
     backtraces than you print.
 
-    Raw backtraces cannot be marshalled. If you need marshalling, you
+    Raw backtraces cannot be marshaled. If you need marshaling, you
```

Comment:
The form "marshalled" seems accepted. Do you have arguments against this specific spelling?